web-animations-2/Overview.bs

```diff
@@ -2063,12 +2129,21 @@ partial dictionary OptionalEffectTiming {
     > until the <a lt='animation effect start time'>start time</a> of any
     > <a>animation effect</a> that may
     > follow, for example, in a <a>sequence effect</a>.
+    > If the <a lt="timeline duration">duration</dfn> of the Animation's
+    > <a>timeline</a> is a percentage, the end delay will be treated as the
+    > result of applying the procedure to convert a
+    > [=time-based animation to a proportional animation=].
```

Comment:
Prefer to defer to a followup PR. The getTiming method returns specified timing, which for now will be time based (or potentially  'auto' for duration).  Once we support CSSNumberish delays, then these interfaces will need to be updated.  At present the start and end delay are not part of CalculatedTiming. Perhaps they should be since they could differ from the specified values after normalization.  Again, prefer to defer.